.github/workflows/run_examples_polynomial.yml

```diff
@@ -86,6 +82,8 @@ jobs:
         ert-storage alembic upgrade head
 
     - name: Start ert-storage
+      env:
```

Comment:
Is the overlap with the above job intentional?

Reply:
Yes. The postgres job wasn't using postgres this entire time because this environment variable wasn't set.

Reply:
Alright! Is it just me or would the jobs benefit from new names? Is really the above job starting Postgres ert storage?